backend/apiPackage.ts

```diff
@@ -67,4 +67,78 @@ export async function getPackagesByName(req : Request, res: Response){
   }catch(error){
     return res.status(500).send(`Error in getPackagesByName: ${error}`);
   }
+}
+
+// Packagedownloadresponsetype
+export type PackageDownloadResponseType = {
```

Comment:
there's already this in the apiSchema.ts file. Use that and remove this exported type.